app/views/govuk_publishing_components/components/_intervention.html.erb

```diff
@@ -1,15 +1,24 @@
 <%= tag.section class: "gem-c-intervention", role: "region" do %>
   <h2 class="gem-c-intervention__title">
-    <a class="govuk-link" href="/next-steps-for-your-business">
-      <%= t("components.intervention.title") %>
-    </a>
+    <a class="govuk-link" href="/next-steps-for-your-business"
+      data-track-category="interventionBanner"
+      data-track-action="interventionClicked"
+      data-track-label="/next-steps-for-your-business"
+      data-track-options="{'dimension29':'<%= t("components.intervention.title") %>'}">
+      <%= t("components.intervention.title") %></a>
   </h2>
 
   <p class="gem-c-intervention__paragraph">
     <%= t("components.intervention.description") %>
   </p>
 
   <p class="gem-c-intervention__paragraph">
-    <%= t("components.intervention.dismiss_html") %>
+    <a class="govuk-link" href="?hide-intervention=true"
+      data-track-category="interventionBanner"
+      data-track-action="interventionDismissed"
+      data-track-label="?hide-intervention=true"
+      data-track-options="{'dimension29':'<%= t("components.intervention.dismiss_link_text") %>'}">
```

Comment:
Looking at other components (for example [here](https://github.com/alphagov/govuk_publishing_components/blob/master/app/views/govuk_publishing_components/components/_step_by_step_nav_related.html.erb#L18)) it seems like there is an option to use the `data-track-dimension` and `data-track-dimension-index` attribute to set custom dimensions which seems a bit cleaner to me, if we're only setting one dimension. 
There's [some additional processing](https://github.com/alphagov/govuk_publishing_components/blob/117837e0079eda0c58a7895dd027279952da8b1a/app/assets/javascripts/govuk_publishing_components/lib/track-click.js#L49) that happens when there are extra options which I think can be avoided in this particular instance.

Reply:
I've made the change